src/serde_arrow_array_utils.erl

```diff
@@ -0,0 +1,82 @@
+%% @private
+%% Utilitary functions for Array Implementations.
+-module(serde_arrow_array_utils).
+-export([validity_bitmap/1]).
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Validity Bitmap & Null Count %%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% This function returns the Validity Bitmap[1] along with its Null Count[2], of
+%% an Array.
+%%
+%% An important thing to consider about our implementation of the Null Count is
+%% that we need to support both `undefined' and `nil' as null values as they are
+%% the conventions for null values in Erlang, and Elixir respectively.
+%%
+%% There are 5 important characteristics to consider about the validity bitmap:
+%%
+%% 1. A null value is represented by a 0 bit, and a non null value by a 1 bit.
+%%
+%% 2. Every 8 elements's validities are batched into a byte, which are then
+%%    reversed as Arrow uses least-significant bit (LSB) numbering (more in
+%%    attached reference).
+%%
+%% 3. If a "batch" consists of less than 8 elements, its validity needs to be
+%%    padded by 0 bits so that it can make a byte.
+%%
+%% 4. Each byte is stored in a slot of a Buffer. This buffer with the validities
+%%    of each batch of 8 elements make up what is called the Validity Bitmap.
+%%
+%% 5. If the Null Count is 0, we can allocate the Validity Bitmap as a NULL
+%%    pointer (which in Erlang's case is `undefined').
+%%
+%% [1]: https://arrow.apache.org/docs/format/Columnar.html#validity-bitmaps
+%% [2]: https://arrow.apache.org/docs/format/Columnar.html#null-count
+
+validity_bitmap(Value) ->
+    case (lists:member(undefined, Value)) or (lists:member(nil, Value)) of
```

Comment:
```suggestion
    case (lists:member(undefined, Value)) orelse (lists:member(nil, Value)) of
```

`orelse` and `andalso` are the short-circuit version of or/and in Erlang.

Reply:
You didn't have to manually change every instance! You just had to tell me to change it. It was just a simple `M-% ) or RET ) orelse RET` (for a single file) or a `sed -i 's/) or/) orelse/g' src/*.erl` (for all files) for me to change it (with some possible change to the regexp)!